src/pal/src/configure.cmake

```diff
@@ -979,6 +957,7 @@ set(SYNCHMGR_SUSPENSION_SAFE_CONDITION_SIGNALING 1)
 set(ERROR_FUNC_FOR_GLOB_HAS_FIXED_PARAMS 1)
 
 list(INSERT CMAKE_REQUIRED_INCLUDES 0 ${CMAKE_CURRENT_SOURCE_DIR}/libunwind/include ${CMAKE_CURRENT_BINARY_DIR}/libunwind/include)
```

Comment:
Shouldn't this be skipped on macOS?

Reply:
I didn't want to include your changes and I was planning to rebase it after your change is merged.

Reply:
Ah. And I was planning to rebase on top of yours :) Let me know what order you want to try and merge these in.

Reply:
Let's get in yours first.